Scripts/Python/xOpeningSequence.py

```diff
@@ -374,7 +393,20 @@ def IFinishStartOrientation(self):
         # start rendering the scene again
         PtEnableRenderScene()
         PtGUICursorOn()
-        OrientationDlg.dialog.show()
+        vault = ptVault()
+        entry = vault.findChronicleEntry(kIntroPlayedChronicle)
+        if entry is not None and IsTutorialPath() and IsCleftSolved():
```

Comment:
Just to double check, but this seems to imply that if you pick the Relto path and haven't completed the cleft, then the orientation GUI will appear.